index.html

```diff
@@ -189,6 +189,27 @@ <h4 class="lang-text">Skills</h4>
 
 </div>
 </section>
+<footer>
+  <p class="contact-description">I'm always interested in hearing about new projects, 
+    so if you'd like to chat please get in touch.</p>
+  <form class="form" action="https://formspree.io/f/xpzbonqw" method="post">
+    <ul>
+      <li><input class="form-input" type="text" placeholder="Full name" maxlength="30" required></li>
```

Comment:
- [ ] According to the project requirements, you need to use a CSS Flexbox property to layout the form elements, please add it.